src/phantom.js

```diff
@@ -75,9 +76,15 @@ export default class Phantom {
         this.process.on('error', error => {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
+            this.kill('Process got an error');
```

Comment:
The message goes `kill` -> `rejectAllCommands` -> `command.deferred.reject` so how does it create a new error? I think it will just end up being `new Error("Process got an error");` right? 

This message should probably something more clear. Like `this.kill('Could not spawn [${phantomjs.path}] executable')`